src/modules/rewards/components/RewardActivities.tsx

```diff
@@ -0,0 +1,103 @@
+import React, { FC } from 'react'
+import { css } from 'styled-components';
+import { Box, Button, Text } from 'blocks';
+
+export type RewardActivitiesProps = {
+  onGetStarted: () => void;
+}
+
+const RewardActivities: FC<RewardActivitiesProps> = ({ onGetStarted }) => {
```

Comment:
```suggestion
const RewardsActivities: FC<RewardsActivitiesProps> = ({ onGetStarted }) => {
```

Also please change the file name